docs/en/compute/Spark.md

```diff
@@ -37,19 +37,22 @@ to the focus of this guide but covered by
 [Unified and Transparent Namespace]({{ '/en/advanced/Namespace-Management.html' | relativize_url }}).
 * Make sure that the Alluxio client jar is available.
 This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
-downloaded from Alluxio [download page](https://www.alluxio.io/download).
+distribution downloaded from Alluxio [download page](https://www.alluxio.io/download).
 Alternatively, advanced users can compile this client jar from the source code
 by following the [instructions]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}).
 
 ## Basic Setup
 
-Distribute the Alluxio client jar across the nodes where Spark drivers or executors are
-running. Specifically, put the client jar on the same local path (e.g. `{{site.ALLUXIO_CLIENT_JAR_PATH}}`) on each node.
+The Alluxio client jar must be distributed across the nodes where Spark drivers or
+executors are running.
+Specifically, put the client jar on the same local path (e.g.
+`{{site.ALLUXIO_CLIENT_JAR_PATH}}`) on each node.
 
-Add the Alluxio client jar to the classpath of Spark drivers and executors
-in order for Spark applications to use the client jar to read and write files in Alluxio.
+The Alluxio client jar must be in the classpath of Spark drivers and executors
+in order for Spark applications to read and write files to Alluxio.
```

Comment:
nit: I typically use "to access Alluxio" to avoid the "read and write from/to" dilemma

Reply:
I like that one